src/arrays/ArrayUtil.java

```diff
@@ -11,85 +11,167 @@
      * Create an array with the given size, fill it with the different values and print it.
      */
     public static void createArray(int size) {
-
+        System.out.println("createArray");
+
+        int[] arr = new int[size];
+        for (int i : arr) {
+            System.out.print(i + ", ");
+        }
+        System.out.println();
+        System.out.println("_______________________");
     }
 
     /**
      * Create two dimensional array with the given sizes, fill it with different values and print it.
      */
     public static void create2dArray(int x, int y) {
+        System.out.println("create2dArray");
```

Comment:
No need to print numbers from 0 to X OR Y, you have to create two dimensional array with X AND Y size.